ui/components/component-library/modal-body/README.mdx

```diff
@@ -0,0 +1,80 @@
+import { Controls, Canvas } from '@storybook/blocks';
+
+import * as ModalBodyStories from './modal-body.stories';
+
+# ModalBody
+
+`ModalBody` is a container component that handles the scrolling of the `Modal` content.
```

Comment:
Adding link to `Modal` page

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/22154/commits/8e21ac564895f55f2eb69bfa7644db9d787e266c


https://github.com/MetaMask/metamask-extension/assets/8112138/9f002fb7-d9d4-4897-b385-62aca820da8a